week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,5 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
```

Comment:
Hey Clement! Reading your code reminded me of the importance of camelCase. You use it really well and consistently and is something I really need to emulate in my own code.

Reply:
@mtbdata711, thanks for your comment, yeah you should as the use of camelCase can help eliminate unnecessary naming error.

Reply:
Nice!

Reply:
that look like a professional coding ;) 
have fun coding